packit/cli/builds/in_image_builder.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import time
+from os import getcwd
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api
+from packit.config import pass_config
+
+logger = logging.getLogger("packit")
+
+
+@click.command("in-image-builder")
+@click.option(
+    "--image-type",
+    default="aws",
+    type=click.STRING,
+)
+@click.option(
+    "--image-architecture",
+    default="x86_64",
+    type=click.STRING,
+)
+@click.option("--nowait", is_flag=True, default=False, help="Don't wait on build")
```

Comment:
```suggestion
@click.option("--wait/--no-wait", default=False, help="Wait for the build to finish")
```

Please also rename it in the header of function to `wait` and the condition to `if wait` (or even better `if not wait: return`) ;)

Reply:
I just copied what we had in some other build method; should we change it there as well?

Reply:
Updated all